t/43-scheduling-and-worker-scalability.t

```diff
@@ -134,7 +134,7 @@ my @job_settings = (
 $job_ids{$jobs->create({@job_settings, TEST => "dummy-$_"})->id} = 1 for 1 .. $job_count;
 
 my $seconds_to_wait_per_worker = 5.0;
-my $seconds_to_wait_per_job    = 1.0;
+my $seconds_to_wait_per_job    = 2.5;
```

Comment:
Is this change based on good reason why this should help or is it just "try to be more robust brute-force"?

Reply:
See https://github.com/os-autoinst/openQA/pull/2885#issuecomment-607115133, I think it really just needs more time. Note that this does not change the polling interval, just the timeout.